package.json

```diff
@@ -45,7 +47,7 @@
     "lint:fix": "eslint . --fix",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "prepare": "husky install",
+    "prepare": "husky",
```

Comment:
Could we please change this to `npx husky` so that it doesn't assume husky is installed? I've done this on toad-reader-server, but forgot to apply it here as well.

Reply:
Thank you, Charles. To make sure we are on the same page, I already enabled GitHub Action on this repository so that this can be tested. If further changes are required, please let me know.